Dockerfile

```diff
@@ -18,11 +18,14 @@ localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 
 ENV LANG en_US.utf8
 
+
```

Comment:
maybe just dont add more emply lines, i guess that is not needed

Reply:
edited the file to remove those for you